frontend/src/cards/ui/OptionMenuIcons.tsx

```diff
@@ -0,0 +1,114 @@
+import ListItemIcon from '@mui/material/ListItemIcon'
+import MenuItem from '@mui/material/MenuItem'
+import {
+    EmailShareButton,
+    FacebookShareButton,
+    LinkedinShareButton,
+    TwitterShareButton,
+    EmailIcon,
+    FacebookIcon,
+    LinkedinIcon,
+    TwitterIcon,
+} from 'react-share'
+import sass from '../../styles/variables.module.scss'
+import { PopoverElements } from '../../utils/hooks/usePopover'
+import styles from './OptionMenuIcons.module.scss'
+
+const shareIconAttributes = {
+    iconFillColor: sass.altDark,
+    bgStyle: { fill: 'none' },
+    size: 39,
+}
+
+interface OptionMenuIconsProps {
+    popover: PopoverElements
+    reportTitle: string;
+}
+
+export default function OptionMenuIcons(props: OptionMenuIconsProps) {
+    let title = `Health Equity Tracker - ${props.reportTitle}`
+    const emailShareBody = `${title}${'\n'}${'\n'}` // Add line breaks here if needed
+    const sharedUrl = window.location.href
```

Comment:
The `sharedUrl` should also include the card `#hash` so that the shared link on social navigates directly to the card that was shared. That prop is already available a level up in the copy link function, so you can just drill it down into this component as well and tack it onto the end of the shared URL

Reply:
The #hash is included in the URL when accessing `window.location.href`. The link takes you directly to the card. I console-logged the URL for each card to be sure. 

> https://deploy-preview-2223--health-equity-tracker.netlify.app/exploredata?mls=1.hiv_care-3.00&group1=All&group2=All**#rate-map**

Reply:
I don't think this is working then; when I click "share to email" the link that is generated in the email doesn't contain a #hash 


https://github.com/SatcherInstitute/health-equity-tracker/assets/41567007/a70f8177-ddc1-42a2-87bc-e0fe9513aeab